lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
+var Promise = require("bluebird");
+var crypto = require("crypto");
+
+var field_type_hashed_text = new Sealious.ChipTypes.FieldType({
+	name: "hashed-text",
+	extends: "text",
+	get_description: function(context, params){
+		return "Hash text with chosen algorithm (default md5)";
```

Comment:
I think we need some better phrasing and more info here. You can actually use the `params` to make the description a bit more verbose:
* what is the minimal length?
* what is the required amount of digits/caps?